pkgs/tools/misc/woeusb-ng/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, python3Packages, fetchFromGitHub, p7zip, parted, grub2 }:
+with python3Packages;
+
+buildPythonApplication rec {
+  pname = "woeusb-ng";
+  version = "0.2.10";
+
+  propagatedBuildInputs = [ p7zip parted grub2 termcolor wxPython_4_0 six ];
+
+  src = fetchFromGitHub {
+    owner = "WoeUSB";
+    repo = "WoeUSB-ng";
+    rev = "v${version}";
+    sha256 = "sha256-Nsdk3SMRzj1fqLrp5Na5V3rRDMcIReL8uDb8K2GQNWI=";
+  };
+
+  postInstall = ''
+    # the maintainer suggested splitting the package into two packages, therefore the gui is removed
```

Comment:
Splitting a package can be done with multiple outputs, whereas this just removes the gui part. Please either include the gui by default, do proper split outputs or update the comment to explain why gui is removed.

Reply:
The initial motivation was to have a cli tool for aarch64. `woeusb` uses `syslinux`, which does not support aarch64.

The gui was never my focus.
The gui itself works with something called "polkit-1/actions". I do not know how to implement it nicely.

Is adding a "TODO" fine? or should I continue the work to include it.

Reply:
Yes, a TODO is fine.